{
  "schema_version": "1.4.0",
  "id": "GHSA-rw75-m7gp-92m3",
  "modified": "2024-09-18T16:07:33Z",
  "published": "2022-05-14T02:09:22Z",
  "aliases": [
    "CVE-2014-0483"
  ],
  "summary": "Django data leakage via querystring manipulation in admin",
  "details": "The administrative interface (contrib.admin) in Django before 1.4.14, 1.5.x before 1.5.9, 1.6.x before 1.6.6, and 1.7 before release candidate 3 does not check if a field represents a relationship between models, which allows remote authenticated users to obtain sensitive information via a to_field parameter in a popup action to an admin change form page, as demonstrated by a `/admin/auth/user/?pop=1&t=password` URI.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:N"
    },
    {
      "type": "CVSS_V4",
      "score": "CVSS:4.0/AV:N/AC:L/AT:N/PR:L/UI:N/VC:L/VI:N/VA:N/SC:N/SI:N/SA:N"
    }
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "Django"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "1.4.14"
            }
          ]
        }
      ]
    },
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "Django"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "1.5"
            },
            {
              "fixed": "1.5.9"
            }
          ]
        }
      ]
    },
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "Django"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "1.6"
            },
            {
              "fixed": "1.6.6"
            }
          ]
        }
      ]
    },
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "Django"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "1.7a1"
            },
            {
              "fixed": "1.7c3"
            }
          ]
        }
      ]
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2014-0483"
    },
    {
      "type": "WEB",
      "url": "https://github.com/django/django/commit/027bd348642007617518379f8b02546abacaa6e0"
    },
    {
      "type": "WEB",
      "url": "https://github.com/django/django/commit/2a446c896e7c814661fb9c4f212b071b2a7fa446"
    },
    {
      "type": "WEB",
      "url": "https://github.com/django/django/commit/2b31342cdf14fc20e07c43d258f1e7334ad664a6"
    },
    {
      "type": "WEB",
      "url": "https://github.com/django/django/commit/f7c494f2506250b8cb5923714360a3642ed63e0f"
    },
    {
      "type": "PACKAGE",
      "url": "https://github.com/django/django"
    },
    {
      "type": "WEB",
      "url": "https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2014-7.yaml"
    },
    {
      "type": "WEB",
      "url": "https://web.archive.org/web/20151016194735/http://secunia.com/advisories/61276"
    },
    {
      "type": "WEB",
      "url": "https://web.archive.org/web/20151016202523/http://secunia.com/advisories/59782"
    },
    {
      "type": "WEB",
      "url": "https://web.archive.org/web/20151023143840/http://secunia.com/advisories/61281"
    },
    {
      "type": "WEB",
      "url": "https://www.djangoproject.com/weblog/2014/aug/20/security"
    },
    {
      "type": "WEB",
      "url": "http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html"
    },
    {
      "type": "WEB",
      "url": "http://www.debian.org/security/2014/dsa-3010"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-200"
    ],
    "severity": "MODERATE",
    "github_reviewed": true,
    "github_reviewed_at": "2024-04-29T14:46:27Z",
    "nvd_published_at": "2014-08-26T14:55:00Z"
  }
}